src/Diagnostics.RuntimeHost/Services/SourceWatcher/Watchers/StorageWatcher.cs

```diff
@@ -191,6 +195,11 @@ private async Task StartBlobDownload(bool startup = false)
                         DiagnosticsETWProvider.Instance.LogAzureStorageWarning(nameof(StorageWatcher), $"No invoker cache exist for {entityType}");
                     }
                 }
+                stopwatch.Stop();
+                if (entitiesToLoad.Count > 0)
+                {
+                    DiagnosticsETWProvider.Instance.LogAzureStorageMessage(nameof(StorageWatcher), $"Blob download complete, startup : {startup.ToString()} time ellapsed {stopwatch.ElapsedMilliseconds} millisecs");
```

Comment:
Possibly log how many entities were loaded during this run?

Reply:
Nice idea